### PR TITLE
Information about the first JLESC Workshop

### PR DESCRIPTION
**Pull request by isabelheisters**
Is it okay like this?
Signed-off-by: Isabel Heisters i.heisters@fz-juelich.de

### CODE REVIEW

File: _events_past/2014-06-09-1st-jlesc-workshop.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+title: 1st JLESC Workshop
+date: 2016-02-19
+updated:
+navbar: Events
+event_location: Sophia Antipolis, France
+event_start: 2014-06-09
+event_end: 2014-06-11
+event_url:
+event_short_url:
+wayback_url:
```
Comment (pancetta) [inline _events_past/2014-06-09-1st-jlesc-workshop.md]: Please add the original URLs here. Also, use the [Web Archive](http://archive.org/web/) to get a quasi-permanent link (this is the `wayback_url`).

File: _events_past/2014-06-09-1st-jlesc-workshop.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+* I/O, Big data
+* Cloud for HPC
+
+
+## Workshop venue
+
+The workshop will take place in the Inria Sophia Antipolis Méditerranée research centre.
+The lodging for the workshop will be at Hotel Mercure and Hotel Ibis.
+They are right next to each other and within reasonable walking distance to the conference venue. 
+
```
Comment (pancetta) [inline _events_past/2014-06-09-1st-jlesc-workshop.md:56]: Could you rephrase the text in past tense? 

> The workshop took place...
> It gathered...

etc.

Comment (isabelheisters) [inline _events_past/2014-06-09-1st-jlesc-workshop.md:56]: The full text or only the Workshop venue?

Comment (pancetta) [inline _events_past/2014-06-09-1st-jlesc-workshop.md:56]: The full text!

File: _events_past/2014-06-09-1st-jlesc-workshop.md (hunk 1 above)
Comment (isabelheisters) [inline _events_past/2014-06-09-1st-jlesc-workshop.md]: So adding the side to the Web archive?

Comment (pancetta) [inline _events_past/2014-06-09-1st-jlesc-workshop.md]: Find the original website using the Web Archive. They have permanent copies of virtually all websites ever created (or close to this), even if they are not online anymore. Check out the site for the 4th workshop!

**Comment by pancetta** [comment]
Very good! I just changed the links a bit (index instead of travel).
